### DISCUSSION TITLE
Schema Support Question

### DISCUSSION THREAD

**Discussion by joezappie** [Automattic/mongoose#11084] [Q&A] [answered]
I found that there was a lot of discussion about JSON schemas and mongoose way back in 2016 and 2017 when Mongo was adding support for it. I can't find anything as of recent on if anything new has come of that. I'm guessing at this point mongoose doesn't support using JSON schema's, but there was some discussion about adding a converter for it. Did that ever go anywhere?

It seems like there's no real solution at the moment to using json schema's with mongoose. There are a lot of 5-7 year old packages on npm for this but none I found are maintained anymore. Very well could be that theres just not a lot of interest for this. Is there a solution to this?

**Comment by vkarpov15** [comment, marked as answer]
Not currently. We may add more support for JSON schemas in the future, but we haven't done any work on that front.
